app/views/users/articles_written.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>user.name 's' Articles</h1>
+<ul>
+  <% user.articles.each do |article| %>
+    <li>
+    <%= link_to article.title, article.link %> - Posted by <%= link_to article.user.name, article.user %> on <%= article.created_at.strftime("%B %d, %Y at %I:%M %p") %>
+    <div>
+      <%= link_to "comment", article %>
+    </div>
+    </li>
+  <% end %>
+</ul>
+
+<%= paginate @articles %>
```

Comment:
This file does not seem to have a corresponding controller method or a route so are you actually using it? If not, delete it. If you are planning on using it, you need to add a controller method and route.

Reply:
Also did not mean to push this.